### ISSUE TITLE
Enable auto-reload of EC2 site from GitHub repo

### ISSUE THREAD

**Issue by dblevins** [microprofile/microprofile-site-config#4]
To get it running: 
- need to get the EC2 tools setup on the jenkins host, 
- get the deployment command to work from the command line there as the jenkin’s process user, 
- then get the CI working, 
- then get it to pay attention to github changes in master

**Comment by exabrial** [comment]
Hey Guys,

So, we've got the site deployment down to one command: `mvn clean package tomee:exec assembly:assembly && eb deploy`. I've updated the README.md with the new site architecture. 

I spent wed, thu, and friday last week trying to find a simple way to build and deploy the site automagically. There were two options, one was to use AWS's code deploy which turned out to be quite complicated and I never got it working right without more concessions. The other, which I'm fairly confident I could get to work, but didn't explore it in depth was to use our Jenkins server to automate the deployments.

I'm going to pause work on this for right now because I have other duties in the Tribe I need to work on, but I think for right now, we don't have a terrible way to make site updates. We can give people outside of the tribe permission to deploy the site by setting up the proper IAM user credentials in AWS if necessary.

**Comment by tbookless** [comment]
Is this documented in the wiki? I see that the readme is updated, but I don't see this maven command.

**Comment by exabrial** [comment]
I do not wish to have the information located in multiple places; should I
put this information on the wiki or should I put it in the readme? I could
also add a link in the wiki to the readme, or vice versa.

**Comment by tbookless** [comment]
I don't think that we are using the same names for things. This is the readme on the wiki that I can see https://github.com/microprofile/microprofile.io/blob/master/README.adoc
 
The maven command is not the same as the description, above.

**Comment by exabrial** [comment]
I see what you're saying now. See the actual site is the in the "Gitpress" project. I'm still confused why this project we're using right now even exists. @tveronezi could you answer this: What is the difference between the `gitpress` project and the `microprofile.io` project, why are there two of them, and can they be combined into one?

**Comment by tveronezi** [comment]
gitpress was misnamed. It should be renamed to 'microprofile.io.website' (or something like that) and 'microprofile.io' should be renamed 'microprofile.io.config' (or something like that)

**Comment by exabrial** [comment]
Returning this to the backlog.The site is on a community EC2 account, it should be easier for anyone in the community to get an account and deploy. Ideally we would still like to do this
